build/docker/Datadog.Trace.ClrProfiler.IntegrationTests.arm64.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/bash
-set -euxo pipefail
+set -uxo pipefail
```

Comment:
why removing `-e`? (it looks especially strange when `-o pipefail` is still here while there is no `|`)
I guess instead of removing `-e` you can simply add `|| true` to a place where you want to ignore the error

the same comment applies to other places

Reply:
We want to grab the logs even if the "main" step fails, but then fail the job _afterwards_. I don't think we can do that with the `|| true` approach 🤔

Reply:
try using `trap`

e.g.
```
cleanup() {
    # ...
}

trap cleanup EXIT
```

Reply:
That has slightly different behaviour to the changes in this though - that will still exit on the first error, as opposed to running all three commands and _then_ exiting. I assume we want the latter approach right @tonyredondo ?

Reply:
Well yeah the intention on this one is to run all 3 dotnet commands no matter what and collect all logs and crashdumps that is generated.

that’s why we were avoiding the early exit.

Reply:
I think it would be better to unset the `-e` before executing the tests: https://superuser.com/a/940504
And then do something like this to check if any of the test failed:  https://stackoverflow.com/a/1000395/859275

Reply:
For simplicity I switched to reenabling `-e` and using `trap` instead. So we'll fail on first error, but I don't think that's a big deal. We copy the logs and the crashdumps in any case